bundles/org.openhab.binding.miio/src/main/resources/database/lumi.curtain.hagl05-miot.json

```
@@ -0,0 +1,202 @@
+{
+	"deviceMapping": {
+		"id": [
+			"lumi.curtain.hagl05"
+		],
+		"propertyMethod": "get_properties",
+		"maxProperties": 1,
+		"channels": [
+			{
+				"property": "fault",
+				"siid": 2,
+				"piid": 1,
+				"friendlyName": "Curtain - Device Fault",
+				"channel": "fault",
+				"channelType": "LumiCurtainHagl05fault",
+				"type": "Number",
+				"refresh": true,
+				"actions": [],
+				"value-list": [
+					{
+						"value": 0,
+						"description": "No faults"
+					}
+				],
+				"readmeComment": "Value mapping [0\u003d\"No faults\"]"
+			},
+			{
+				"property": "current-position",
+				"siid": 2,
+				"piid": 3,
+				"friendlyName": "Curtain - Current Position",
+				"channel": "current-position",
+				"channelType": "miot_uint8",
+				"type": "Number",
+				"refresh": true,
+				"actions": []
+			},
+			{
+				"property": "status",
+				"siid": 2,
+				"piid": 6,
+				"friendlyName": "Curtain - Status",
+				"channel": "status",
+				"channelType": "LumiCurtainHagl05status",
```

Looks like you need to double check the channel types names you are using. I'm sure that others need to be fixed as well.
```suggestion
				"channelType": "LumiCurtainHagl05_status",
```

---

darn, indeed is a bug in my converter utility. 
Made the improvement to have the options from the spec but got indeed mismatch in the types formatting.